ramls/examples/loan.json

```diff
@@ -5,7 +5,7 @@
   "itemId": "cb20f34f-b773-462f-a091-b233cc96b9e6",
   "loanDate": "2017-03-01T23:11:00-01:00",
   "dueDate": "2017-03-15T23:11:00-01:00",
-  "itemEffectiveLocationAtCheckOut": "07f83048-6737-4c6a-a3bc-c49cd9beaa38",
+  "itemEffectiveLocationIdAtCheckOut": "07f83048-6737-4c6a-a3bc-c49cd9beaa38",
```

Comment:
This is a breaking compatibility change, the `loan-storage` interface will need changing to version `7.0` and all dependent modules will need updating for compatibility.